Towerise/ClientApp/app/components/play-space/play-space.component.css

```diff
@@ -0,0 +1,9 @@
+﻿svg text {
```

Comment:
it's standard in angular 2 apps to use scss files, so just change the extension.